Makefile

```diff
@@ -132,6 +134,8 @@ docker-push:
 
 release:
 	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
+
+release-deploy:
```

Comment:
This sounds more like `build-release-manifest`

Reply:
We don't `build` here as we only update manifests with released tags so maybe `release-manifest`?

Reply:
SGTM